python/cog/server/helpers.py

```diff
@@ -1,28 +1,38 @@
+from __future__ import annotations
```

Comment:
Did we? It doesn't solve all problems as it only changes behavior for annotations (and not for types used in inheritance, for example) but it allows `Foo | None` rather than `Optional[Foo]` so I'm inclined to start using it more widely.

Reply:
I think I had test failures when using it that went away when I removed it before. 🤷

Reply:
I think we were trying to use it to solve a problem it can't solve, namely allow `Future[Done]` in some non-annotation context.

Reply:
Ah, possibly.